stock_account_move_reset_to_draft/tests/test_stock_account_move_reset_to_draft.py

```diff
@@ -106,67 +121,65 @@ def test_purchase_order_flow_01(self):
     @mute_logger("odoo.models.unlink")
     def test_purchase_order_flow_02(self):
         # PO for a product: 2 pcs at EUR10
-        order_form = Form(self.env["purchase.order"])
-        order_form.partner_id = self.partner
-        with order_form.order_line.new() as line_form:
-            line_form.product_id = self.product
-            line_form.price_unit = 10
```

Comment:
This was misleading as `price_unit` would be reset to `0` with the next `product_qty` update step.

Reply:
Yes, you are right.